internal/services/cragcast/cragcast.go

```diff
@@ -0,0 +1,227 @@
+// this will contain functions for different forecast functionality.
+// eg. GetForecast, GetHumidity, GetRainfall, etc...(not necessarily these, just giving them as examples to illustrate)
```

Comment:
If we keep this new package `cragcast` (see [comment here](https://github.com/dsauerbrun/cragcast/pull/9/files#r1181073623)), this is a note to clean up these docs as they will show up as the official package docs in GoDoc.

Reply:
do i just put the comments below `package cragcast` so they dont show up in the docs?